src/ImageGallery.jsx

```diff
@@ -63,7 +63,7 @@ export default class ImageGallery extends React.Component {
     renderRightNav: React.PropTypes.func,
     renderPlayPauseButton: React.PropTypes.func,
     renderFullscreenButton: React.PropTypes.func,
-    renderItem: React.PropTypes.func,
+    renderItem: React.PropTypes.func
```

Comment:
You can leave the trailing commas (they are intended) as they are handled by the transform